AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/AWSDataStorePlugin+DataStoreSubscribeBehavior.swift

```diff
@@ -11,10 +11,27 @@ import Combine
 extension AWSDataStorePlugin: DataStoreSubscribeBehavior {
     @available(iOS 13.0, *)
     public func publisher<M: Model>(for modelType: M.Type)
+        -> AnyPublisher<MutationEvent, DataStoreError> {
+        return publisher(for: modelType.modelName)
+    }
+
+    @available(iOS 13.0, *)
+    public func publisher(for modelName: ModelName)
+        -> AnyPublisher<MutationEvent, DataStoreError> {
+            reinitStorageEngineIfNeeded()
+            // Force-unwrapping: The optional 'dataStorePublisher' is expected
+            // to exist for deployment targets >=iOS13.0
+            return publisher()
+                .filter { $0.modelName == modelName }
+                .eraseToAnyPublisher()
+    }
+
+    @available(iOS 13.0, *)
+    public func publisher()
```

Comment:
I don't know whether availability checks work for computed properties, but if they do, try modeling this as a computed property rather than a method.

Reply:
Moved to computed property